server/superdesk/__init__.py

```diff
@@ -113,3 +113,20 @@ def register_default_user_preference(preference_name, preference):
 
 def register_default_session_preference(preference_name, preference):
     default_session_preferences[preference_name] = preference
+
+
+def register_resource(name, resource, service, backend=None, privilege=None):
```

Comment:
`service` can default to `None` as well, in that case use the default Service, we have plenty of resources configured with it.